featurebyte/service/tile/tile_task_executor.py

```diff
@@ -79,13 +85,11 @@ async def execute(self, session: BaseSession, params: TileScheduledJobParameters
         monitor_tile_start_ts_str = tile_start_ts_str
 
         # use the last_tile_start_date from tile registry as tile_start_ts_str if it is earlier than tile_start_ts_str
-        registry_df = await retry_sql(
-            session,
-            f"SELECT LAST_TILE_START_DATE_ONLINE FROM TILE_REGISTRY WHERE TILE_ID = '{params.tile_id}' AND LAST_TILE_START_DATE_ONLINE IS NOT NULL",
```

Comment:
Note: this was wrong. The last tile date should be retrieved by looking up both `TILE_ID` and `AGGREGATION_ID`.